src/components/CollectionItem.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "styled-components";
+
+const ImageContainer = styled.img`
+  width: 88px;
+  height: 88px;
+  border: 4px solid white;
+  object-fit: cover;
+  object-position: center center;
+  border-radius: 15px;
+  box-shadow: 0px 3px 6px #00000037;
```

Comment:
It's the XD value for box-shadow, right? I used the same, but Marwin recommended using rgba instead...

Reply:
Yes thank you! I've already changed this in my other Pull Request... sorry for the confusion